src/MSAL.PCL/Internal/Cache/TokenCacheAccessor.cs

```diff
@@ -70,10 +70,15 @@ public void DeleteRefreshToken(RefreshTokenCacheItem refreshToken‪Item)
         {
             TokenCachePlugin.DeleteRefreshToken(refreshToken‪Item.GetTokenCacheKey().ToString());
         }
+        
+        public ICollection<string> GetAllAccessTokensAsString()
```

Comment:
Is there a reason for returning an ICollection here and an IList below? Seems a bit inconsistent.